app/services/data_migrations/set_institution_country_to_gb_on_uk_degrees.rb

```diff
@@ -0,0 +1,20 @@
+module DataMigrations
+  class SetInstitutionCountryToGbOnUkDegrees
+    TIMESTAMP = 20240111093629
+    MANUAL_RUN = false
```

Comment:
could we set this to `MANUAL_RUN = true` and also batch the update,  since it's ~180k records?

Reply:
Discussing offline but adding this for completion: https://github.com/DFE-Digital/apply-for-teacher-training/pull/8978/commits/cf8a4e409be097c243388bbca0e0764a5fba76d0